Codeinwp/neve
# Fix text color not configurable with menu skin mode 2

### Summary
The text color was not configurable if you choose the second skin mode. I've added a new color control that appears when you select the second skin mode.

### Will affect the visual aspect of the product
NO

### Test instructions
- Add this PR, add a primary menu component and choose the second `Hover Skin Mode`
- You should see another color named `Items Hover Text Color` appearing 
- Change that color and see if it works well
- Change the hover skin mode and see if that color is still applying


Closes #3653.
